README.md

```diff
@@ -5,6 +5,19 @@
 
 ## Ingester
 
+Requirements:
+1. go
+    ```
+    wget https://dl.google.com/go/go1.11.5.linux-amd64.tar.gz
+    sudo tar -C /usr/local/ -xzf go1.11.5.linux-amd64.tar.gz
+    export PATH=$PATH:/usr/local/go/bin
+    source $HOME/.profile
```

Comment:
I think we should link to the [official installation instructions](https://golang.org/doc/install). The exporting of the `PATH` variable here doesn't actually affect the `$HOME/.profile` file anyway. You would actually need to add the line to the `.profile` file